src/main/scala/org/aphreet/c3/lib/FileUpload.scala

```diff
@@ -59,24 +59,30 @@ object FileUpload extends RestHelper with C3Loggable{
 
   serve {
     case "upload" :: "file" :: currentPath Post req => {
-      withCurrentUserAndGroupCtx(req, currentPath){
+      withCurrentUserAndGroupCtx(req, currentPath) {
 
-          val uploads = req.uploadedFiles
-          logger.info("Uploaded files: " + uploads)
+        val uploads = req.uploadedFiles
+        logger.info("Uploaded files: " + uploads)
 
-          val filePath = c3FilePath(currentPath)
+        val filePath = c3FilePath(currentPath)
 
-          logger.info("Path to upload: " + filePath)
-          userGroupIds: UserGroupIds => {
+        logger.info("Path to upload: " + filePath)
+        userGroupIds: UserGroupIds => {
 
-          try{
+          try {
             val ojv: List[JObject] = uploads.map { fph =>
               val url = removeTrailingIndex(currentPath).mkString("/", "/", "/") + fph.fileName
-              val description = req.param(s"description_${fph.fileName}") match {case Full(temp) => temp; case _ => ""}
+              val description = req.param(s"description_${fph.fileName}") match {
+                case Full(temp) => temp;
+                case _ => ""
```

Comment:
Description is necessary field for filling, in this case we should throw warning msg.